client/my-sites/plugins-wpcom/jetpack-plugins.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+const jetpackPlugins = [
+	{
+		category: 'engagement',
+		name: translate( 'Engagement' ),
```

Comment:
this is unfortunately not going to work. these values are assigned once when the module loads, but we want to update translations dynamically from within the app. to do this, we have to call `translate()` from within `render()` or in the function of a stateless functional component.

a way I have gotten around this is by passing in the translate function and calling it as seen in a comment above

```js
const jetpackPlugins = translate => [
	{
		category: 'engagement',
		name: translate( 'Engagement' ),
		…
	},
	…
];

const plugins = jetpackPlugins( this.translate );
```